test/db-migrations/migrator.js

```diff
@@ -0,0 +1,101 @@
+module.exports = {
+  exists,
+  hasRun,
+  runBefore,
+  runIncluding,
+  restoreMigrations,
+};
+
+const fs = require('node:fs');
+const { execSync } = require('node:child_process');
+
+// Horrible hacks.  Without this:
+//
+// 1. production migration code needs modifying, and
+// 2. it takes 3 mins+ just to run the migrations
```

Comment:
I don't think migrations take 3 minutes to run, do they?

Reply:
Running each migration in turn with knex takes 3 minutes.  Maybe the comment is not helpful at this point.